DockerfileDocs

```diff
@@ -0,0 +1,9 @@
+FROM debian:latest
```

Comment:
I suggest to use a specific version here. It's unlikely that debian is going to break the things below in a next major release, but it makes sure it doesn't in case it does.